conjur/policy/policy_controller.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+PolicyController module
+
+This module is the controller that facilitates all list actions
+required to successfully execute the POLICY command
+"""
+import sys
+
+
+class PolicyController:
+    """
+    PolicyController
+
+    This class represents the Presentation Layer for the POLICY command
+    """
+    def __init__(self, ssl_verify, policy_logic, policy_data):
+        self.ssl_verify = ssl_verify
+        self.policy_logic = policy_logic
+        self.policy_data = policy_data
+
+    def load(self):
+        result = self.policy_logic.run_action(self.policy_data)
+        self.print_json_result(result)
+
+    @classmethod
+    def print_json_result(cls, result):
+        """
+        Method to print the JSON of the returned result
+        """
+        sys.stdout.write(result+'\n')
```

Comment:
where it converts it to json?

Reply:
check out variable_logic.py, I return the already decoded/json object

Reply:
updating to `print_response` because it won't necessarily return a json

Reply:
yes but the function dont convert it to json it just sys.stdout.write so make rename function to print_to_stdout ?

Reply:
removed this method because it is just writing to STDOUT

Reply:
not sure whats happening here.. 
BTW suggest flush the buffer after writing to stdout. "sys.stdout.flush()" but no biggy if it a small str